src/katsdpcontroller/generator.py

```diff
@@ -1172,6 +1170,15 @@ def get_baseline_index(a1, a2):
                 "{ports[aioconsole]}",
             ]
         )
+        output_config = {
+            "name": escape_format(stream.name),
+            "heap_accumulation_threshold": str(round(stream.int_time / heap_time)),
```

Comment:
```suggestion
            "heap_accumulation_threshold": round(stream.int_time / heap_time),
```
The `str` isn't needed because the `f"{key}=value"` lower down will stringify it.